src/pages/guide/native/convert-from-ocaml.md

```diff
@@ -3,11 +3,11 @@ title: Converting from OCaml
 order: 4
 ---
 
-Since Reason's just another syntax for OCaml, convert an OCaml project over is straightforward and doesn't require semantic changes. However, there are a few build setup & miscellaneous changes.
+Since Reason is just another syntax for OCaml, converting an OCaml project over is straightforward and doesn't require semantic changes. However, there are a few build setup and miscellaneous changes required.
```

Comment:
Minor grammar fixes. "Reason's" is technically correct, but using contractions with proper nouns is less common and it might trip up readers for which English is a second language.